docs/optimization.md

```diff
@@ -161,7 +161,7 @@ import nevergrad as ng
 from nevergrad.functions import MultiobjectiveFunction
 import numpy as np
 
-f = MultiobjectiveFunction(multiobjective_function=lambda x: np.sum(x**2) , upper_bounds=[2.5, 2.5])
+f = MultiobjectiveFunction(multiobjective_function=lambda x: [sum(x**2), sum((x-1)**2)], upper_bounds=[2.5, 2.5])
```

Comment:
sum() will be much slower if x is a numpy array.

Reply:
good catch, thx!